resources/js/components/twc-ui/checkbox.tsx

```
@@ -49,13 +49,13 @@ const BaseCheckbox = ({ className, children, ...props }: AriaCheckboxProps) => (
           className={cn(
             'flex size-4 shrink-0 items-center justify-center rounded-sm border border-input text-base text-current',
             /* Focus Visible */
-            "group-data-[focus-visible]/checkbox:outline-none group-data-[focus-visible]/checkbox:ring-1 group-data-[focus-visible]/checkbox:ring-ring",
+            "group-data-focus-visible/checkbox:outline-none group-data-focus-visible/checkbox:ring-1 group-data-focus-visible/checkbox:ring-ring",
             /* Selected */
-            'group-data-[indeterminate]/checkbox:bg-primary group-data-[selected]/checkbox:bg-primary group-data-[indeterminate]/checkbox:text-primary-foreground group-data-[selected]/checkbox:text-primary-foreground',
+            'group-data-indeterminate/checkbox:bg-primary group-selected/checkbox:bg-primary group-data-indeterminate/checkbox:text-primary-foreground group-selected/checkbox:text-primary-foreground',
```

Biome (2.1.2)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)